_scripts/mkindex.py

```diff
@@ -115,7 +115,7 @@ def index_grammars(root : str) -> Sequence[dict]:
                 'lexer' : lexer,
                 'parser' : parser,
                 'start' : get_single_value("entryPoint", pom),
-                'example' : examplesFiles[0]
```

Comment:
I thought I saw an example where there were multiple examples? And I think it might always come back with a list anyway.

Reply:
Right, it should be a list of files, not just one file. Otherwise why offer a selection for input files?

But, something didn't work because there's no grammars.json file generated.  Checking my repo copy to see where the workflow doesn't work.

Reply:
Haha oh right. Duh. Right.